backend/small_eod/tags/urls.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import routers
+from django.urls import path, include
+
+from ..users.views import UserViewSet
+from ..institutions.views import AddressDataViewSet, InstitutionViewSet
+from ..cases.views import CaseViewSet
+from ..dictionaries.views import DictionaryViewSet
+router = routers.DefaultRouter()
+router.register('users', UserViewSet)
+router.register('address', AddressDataViewSet)
```

Comment:
Dlaczego rejestrujesz także ```AddressDataViewSet```?

Reply:
To sa tylko moje testy, chcialem zobaczyc jak sie zachowuja viewsety na czyms co nie ma zadnych wymaganych relacji.